src/tools/pontoon/teams_projects.md

```diff
@@ -22,22 +22,23 @@ The Teams page lists all locales that are enabled in Pontoon. It can be reached
 
 ### Team page
 
-Access a specific team’s page by selecting it from the Teams page or by appending a locale code to the end of the Pontoon base URL with `/{LOCALE}` (e.g [pontoon.mozilla.org/it](https://pontoon.mozilla.org/it) for Italian). This page shows a list of all projects enabled for the locale. Clicking on one of these projects leads to the [Localization page](#localization-page).
+Access a specific team’s page by selecting it from the Teams page or by appending a locale code to the end of the Pontoon base URL with `/{LOCALE}` (e.g [pontoon.mozilla.org/it](https://pontoon.mozilla.org/it) for Italian).
 
 The page header contains linguistic information about the current locale (plural form, writing direction, etc.), plus an indication of the overall status of completion and statistics. The statistics displayed in the rightmost column are active links to access the Localization page for all projects. For example, select `MISSING` to see all missing strings across projects enabled for that locale in Pontoon.
 
 ![Header of team page for Italian](../../assets/images/pontoon/teams_projects/team_page_header.png)
 
-There are up to 5 tabs available to all [roles](users.md#user-roles), and 1 additional tab for those with Administrator or Team manager roles:
+Underneath the header, the following tabs are available to navigate between various Team page panels:
 
 * **Projects**: a list of all projects enabled in Pontoon for this locale.
 * **Contributors**: a list of active contributors with their statistics, filterable by time (all time, last 12/6/3/1 months).
-* [**Insights**](#insights-graphs): contains data and trends presented in a graphical format about active users, time to review suggestions, review activity, and translation activity.
-* **Bugs**: a list of open bugs for this locale, retrieved from [Bugzilla](https://bugzilla.mozilla.org/). Note this is specific to Mozilla’s deployment of Pontoon.
+* [**Insights**](#insights-graphs): data and trends on contributor and localization activity.
+* **Bugs**: a list of open bugs for the locale, retrieved from [Bugzilla](https://bugzilla.mozilla.org/) (available in Mozilla's deployment).
 * **Info**: information about the team. Team managers can edit this by clicking on the `EDIT` button.
-* **Permissions**: [manage user permissions](users.md#managing-permissions).
+* **Permissions**: [permissions management panel](users.md#managing-permissions) (available to [Team Managers](users.md#user-roles)).
+* **TM**: [translation memory management](#translation-memory-management) panel (available to [Translators](users.md#user-roles)).
```

Comment:
Would this not also be available to Team Managers as well?

Reply:
Right, as "Team Managers can perform the same actions as translators" ([docs](https://mozilla-l10n.github.io/localizer-documentation/tools/pontoon/users.html#user-roles)). Feel free to suggest a change if you think we should make that more explicit.

Reply:
```suggestion
* **TM**: [translation memory management](#translation-memory-management) panel (available to [Translators](users.md#user-roles) and [Team Managers](users.md#user-roles)).
```

Reply:
@mathjazz - Made a suggestion.